docs/source/faq.rst

```diff
@@ -0,0 +1,113 @@
+SSL parameter to appropriate config files.
+
+For MDS it can be set by editing ``~/.motech/config/datanucleus.properties`` and changing the property ``javax.jdo.option.ConnectionURL``.
+For quartz you have to edit the variable ``org.quartz.dataSource.motechDS.URL`` in ``quartz.properties``. If your config
+source is FILE, then you have to edit the file in scheduler directory in ``~/.motech/config``. If your source is UI, you
+have to change the quartz configuration through the manage modules section in the Admin UI.
+
+MOTECH is not working as expected, how can I check what's wrong?
+----------------------------------------------------------------
+All the problems that MOTECH encounters are logged. The logs can be checked in two ways. If MOTECH is working you can
+access logs by choosing ``Server Log`` from ``Admin`` panel on the UI. There you can also specify logging level for
+specific packages - this is done by selecting wanted log level in ``Server Log -> Log options`` panel.
+
+If access to UI is not possible, for standard MOTECH installation on Tomcat logs can be also found in ``catalina.out``
+file located in Tomcat installation logs folder. For example for MOTECH deployed on Tomcat 7.0.62 the ``catalina.out``
+is located in ``~/apache-tomcat-7.0.62/logs`` folder. The content of the log file can be displayed by any text editor
+or in Ubuntu console by for example ``less catalina.out`` command.
+
+I am getting blueprint timeout error - what does it mean?
+---------------------------------------------------------
+Blueprint timeout error is encountered when one of the modules has reference to the service from another module defined in
+``blueprint.xml`` file and that service has not been made available. By default module waits 5 minutes for required service,
+if in this time service won't be delivered the timeout error will be thrown. First step in resolving blueprint timeout
+errors should be checking if the problematic service is correctly registered as OSGi service. Below you can see an example
+of properly defined OSGi service exporting:
+
+.. code-block:: xml
+
+    <osgi:service ref="subscriptionService"
+            interface="org.motechproject.hub.service.SubscriptionService"
+            auto-export="interfaces" />
+
+and importing:
+
+.. code-block:: xml
+
+    <osgi:reference id="hubTopicMDSService"
+          interface="org.motechproject.hub.mds.service.HubTopicMDSService" />
+
+If service is defined correctly and blueprint timeout error is still encountered it probably means that module which
+should deliver the service didn't start and further investigation for origin error cause should be taken at that module.
+
+How to create a web-service with MOTECH?
+----------------------------------------
+Creating web-service with MOTECH is done by using Spring Web Services. After adding all required dependencies for
```

Comment:
can be done using the Spring Web Services framework (link to Spring WS), which is the suggested approach.